master/internal/command/tensorboard_manager.go

```diff
@@ -232,6 +232,21 @@ func (t *tensorboardManager) newTensorBoard(
 
 			logBasePath = "s3://" + c.Bucket()
 
+		case expconf.AzureConfig:
+			if connectionString := c.ConnectionString(); connectionString != nil {
+				uniqEnvVars["AZURE_CONNECTION_STRING"] = *connectionString
+			}
+			if accountURL := c.AccountURL(); accountURL != nil {
+				uniqEnvVars["AZURE_ACCOUNT_URL"] = *accountURL
+			}
+			if credential := c.Credential(); credential != nil {
+				uniqEnvVars["AZURE_CREDENTIAL"] = *credential
+			}
+
+			uniqEnvVars["AZURE_CONTAINER"] = c.Container()
```

Comment:
where/how are these env variables being used?

Reply:
They aren't used anywhere. I included them for the sake of completeness. Should I remove them?

Reply:
AWS variables above can be used by `boto`, I don't think azure python lib will use these. 
and GCS code below doesn't have any, so there's no completeness here really.

Reply:
sounds good, will delete!